perllib/FixMyStreet/Cobrand/Hackney.pm

```diff
@@ -243,6 +243,13 @@ sub munge_sendreport_params {
     }
 }
 
+sub open311_munge_update_params {
```

Comment:
I was going to say there is a FixMyStreet::Roles::Open311Multi you can include, but I guess that doesn't work unless Hackney had a function already, hmm. Perhaps the role function could be a normal function that calls super instead? Doesn't seem to need to be an "around" way of doing it, I mean.